eplaunch/runner.py

```diff
@@ -2,6 +2,12 @@
 
 from eplaunch.interface.application import EpLaunchApplication
 
-gettext.install("app")  # replace with the appropriate catalog name
-app = EpLaunchApplication(0)
-app.MainLoop()
+
+def main():
+    gettext.install("app")  # replace with the appropriate catalog name
+    app = EpLaunchApplication(0)
+    app.MainLoop()
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
Added a `main()` function which can be added to the Python packaging `entry_points` keyword.  Then added a call to this main function inside here so it should result in no change in behavior.